### PR TITLE
Fix typo in build.ps1 and use nuget.config

### PR DESCRIPTION
**Pull request by AArnott**

### CODE REVIEW

File: Build/build.ps1 [hunk 1]
```diff
@@ -193,12 +193,13 @@ function EnsureNuGetExists()
...
 }
 
-function EnsureNuGetPacakge($packageName, $packagePath, $packageVersion)
+function EnsureNuGetPackage($packageName, $packagePath, $packageVersion)
 {
   if (!(Test-Path $packagePath))
   {
     Write-Host "Couldn't find $packagePath. Downloading with NuGet"
-    exec { & $nugetPath install $packageName -OutputDirectory $buildDir\Temp -Version $packageVersion | Out-Default } "Error restoring $packagePath"
+    $configFilePath = Resolve-Path "$PSScriptRoot\..\Src\nuget.config"
```
Comment (JamesNK) [inline Build/build.ps1]: What about just `"$sourceDir\nuget.config"`

**Comment by JamesNK** [comment]
Was this being missing causing any issues?

**Comment by AArnott** [comment]
> Was this being missing causing any issues?

Yes. On my machine I have nuget package sources added machine-wide that require authentication. So your package install that runs from your runbuild script failed because no auth token was available.

File: Build/build.ps1 (hunk 1 above)
Comment (AArnott) [reply Build/build.ps1]: Ah, good idea. Done.

**Comment by JamesNK** [comment]
Okie dokie

**Pull request merged**